redirects.yaml

```diff
@@ -12,3 +12,8 @@ operator-services/?: /about
 ops-code-quality/?: /about
 overview/?: /about
 universal-operators/?: /about
+docs/lma2/on MicroK8s/?: https://charmhub.io/topics/canonical-observability-stack/install/microk8s
+docs/lma2/install/microk8s/?: https://charmhub.io/topics/canonical-observability-stack/install/microk8s
+docs/lma2/juju-topology/?: https://charmhub.io/topics/canonical-observability-stack/juju-topology
+docs/lma2/?: https://charmhub.io/topics/canonical-observability-stack
+docs/cos/?: https://charmhub.io/topics/canonical-observability-stack
```

Comment:
Do you need to redirect as well `/docs/cos/`? In your previous PR, you changed the name from `lm2` to `cos`

Reply:
I find no evidence of links pointing directly to /docs/cos, but sure, why not

Reply:
@jkfran is there a way to redirect an entire subpath? E.g., `s|docs/lma2|https://charmhub.io/topics/canonical-observability-stack|`

Reply:
Oh I like!